automation/src/test/java/framework/DriverManagerFactory.java

```
@@ -0,0 +1,17 @@
+package framework;
+
+public class DriverManagerFactory extends TestBase {
+	
+	public static DriverManager getManager(String browserType) {
+        DriverManager driverManager = null;
+
+		if (browserType.equals("chrome")) {
+			driverManager = new ChromeDriverManager();
+		} else if (browserType.equals("edge")) {
+			driverManager = new EdgeDriverManager();
+		}
+		

```

You should throw an exception if neither of these browserTypes are supported.